bundles/org.openhab.core.voice/src/test/java/org/openhab/core/voice/internal/cache/AudioStreamCacheWrapperTest.java

```diff
@@ -0,0 +1,115 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.voice.internal.cache;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.Mockito.*;
+
+import java.io.IOException;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.openhab.core.audio.AudioStream;
+import org.openhab.core.voice.TTSException;
+
+/**
+ * Test the wrapper stream in the cache system
+ *
+ * @author Gwendal Roulleau - Initial contribution
+ */
+@NonNullByDefault
+@ExtendWith(MockitoExtension.class)
+public class AudioStreamCacheWrapperTest {
+
+    TTSResult mockedTTSResult = Mockito.mock(TTSResult.class);
```

Comment:
```suggestion
    private @Mock @NonNullByDefault({}) TTSResult ttsResultMock;
```

Mockito can do the mocking by itself, no need to call `mock`. We usually name mocked objects `<name>Mock` in core. For consistency, please change that (also below, and inline the null-annotation).

Reply:
Done, thanks for the tip.